pkg/koordlet/qosmanager/plugins/cpuevict/cpu_evict.go

```diff
@@ -326,6 +357,20 @@ func (c *cpuEvictor) getPodEvictInfoAndSort() []*podEvictCPUInfo {
 	return bePodInfos
 }
 
+func (c *cpuEvictor) getBEMilliAllocatable() float64 {
+	node := c.statesInformer.GetNode()
+	if node == nil || node.Status.Allocatable == nil {
+		return -1
+	}
+
+	batchCPUQuant, ok := node.Status.Allocatable[apiext.BatchCPU]
```

Comment:
use `GetBatchMilliCPUFromResourceList`

Reply:
It may cause unexpected allocatable like `cpu`.

Reply:
need a todo on internal branch